packages/storybook/assets/css/gallery.css

```diff
@@ -0,0 +1,50 @@
+.sbdocs-content.sbdocs-content {
+  position: absolute;
+  inset: 0;
+  max-width: none;
+  padding: 3rem;
+  box-sizing: border-box;
+}
+
+.sbdocs-content + div {
+  display: none;
+}
+
+#ui-components {
+  margin-bottom: 3rem;
+}
+
+.gallery--items {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(450px, 1fr));
+  gap: 2rem;
+}
+
+.gallery--item {
+  display: flex;
+  flex-flow: column;
+  gap: 1rem;
+  padding-bottom: 1rem;
+}
+
+.gallery--item-title.gallery--item-title {
+  text-align: center;
+  font-size: 1.25rem;
+}
+
+.gallery--item-title > p {
+  margin: .5rem 0;
+}
+
+.sbdocs-preview.sbdocs-preview {
+  margin: 0;
+  flex: 1 1 auto;
+  box-shadow: none;
+  border: none;
+  border-radius: .5rem;
+  max-height: 150px;
```

Comment:
this line cause trouble on the technical documentation 
![image](https://github.com/user-attachments/assets/5b0b4e5a-0a54-4544-927a-a6aa10ec0859)